### PR TITLE
Fix $post_int init

### PR DESCRIPTION
**Pull request by cojennin**
Doing some minor cleanup on @jeremyclarke's PR #328 per @mjangda's comments.

@jeremyclarke @mjangda do you have a minute to review?

### CODE REVIEW

File: common/php/class-module.php [hunk 1]
```diff
@@ -253,27 +253,25 @@ function enqueue_datepicker_resources() {
 	function get_current_post_type() {
 		global $post, $typenow, $pagenow, $current_screen;
 		//get_post() needs a variable
-		$post_int;
-		if( isset( $_REQUEST['post'] ) )
-			$post_int = (int)$_REQUEST['post'];
+		$post_int = isset( $_REQUEST['post'] ) ? (int)$_REQUEST['post'] : false;
```
Comment (mjangda) [inline common/php/class-module.php]: Minor: it'd be nice to rename the variable to `$post_id`.

File: tests/test-class-module.php [hunk 2]
```diff
@@ -44,6 +44,44 @@ function test_add_caps_to_role() {
...
+
+		unset( $_REQUEST['post_type'] );
+
+		set_current_screen( 'edit.php' );		
+
+		$this->assertEquals( 'post', $EditFlowModule->get_current_post_type() );
+
+		_unregister_post_type( 'content' );
+
+		set_current_screen( 'front' );
```
Comment (mjangda) [inline tests/test-class-module.php]: These should probably all be separate tests?

File: tests/test-class-module.php [hunk 3]
```diff
@@ -1,10 +1,13 @@
 <?php
 
 class WP_Test_Edit_Flow_Class_Module extends WP_UnitTestCase {
+
 	protected static $admin_user_id;
+	protected static $EditFlowModule;
 
 	public static function wpSetUpBeforeClass( $factory ) {
 		self::$admin_user_id = $factory->user->create( array( 'role' => 'administrator' ) );
+		self::$EditFlowModule = new EF_Module();
```
Comment (mjangda) [inline tests/test-class-module.php]: We should probably instantiate a new module for each test (i.e. in `setUp`) and de-init in `tearDown` (i.e. `self::$EditFlowModule = null`).

**Comment by mjangda** [comment]
Nice! One more note and then 🚢

**Comment by cojennin** [comment]
@mjangda made the tweaks. Split the tests and did some cleanup

File: tests/test-class-module.php (hunk 3 above)
Comment (cojennin) [inline tests/test-class-module.php]: Is `self::$EditFlowModule = null` necessary in the tearDown? (Just curious, don't know much about PHP GC)

Comment (mjangda) [inline tests/test-class-module.php]: Not strictly necessary, I guess, but it's a good idea to clean up after ourselves (we'd end up with reference after the last test is run).

Comment (cojennin) [inline tests/test-class-module.php]: Gotcha, added.

**Comment by mjangda** [comment]
🚢 🚢 🚢  (once Travis finishes)

**Comment by cojennin** [comment]
Tests look good, merging.

Thanks @jeremyclarke for kicking this off with #328

**Comment by jeremyclarke** [comment]
Thanks guys and sorry I didn't get to it!

**Pull request merged**